src/deriv_api/DerivAPIBasic.js

```diff
@@ -76,6 +76,8 @@ export default class DerivAPIBasic extends DerivAPICalls {
         this.pendingRequests       = {};
         this.expect_response_types = {};
         this.subscription_manager  = new SubscriptionManager(this);
+        this.reconnect_timeout     = '';
+        this.keep_alive_interval   = '';
```

Comment:
I don't know, I just want to discuss:
Will `false` be better than `''`
```suggestion
        this.reconnect_timeout     = false;
        this.keep_alive_interval   = false;
```